Dockerfile

```diff
@@ -18,12 +18,7 @@
 # granted to it by virtue of its status as an Intergovernmental Organization or
 # submit itself to any jurisdiction.
 
-FROM ubuntu:16.04
-RUN apt-get update
-RUN apt-get -y upgrade
-RUN apt-get -y install rabbitmq-server
-RUN apt-get -y autoremove && apt-get -y clean
-RUN service rabbitmq-server start
+FROM rabbitmq:3.6-alpine
 ADD start.sh /start.sh
```

Comment:
Is `start.sh` still necessary after moving to official rabbitmq image?  I guess it shouldn't be...